products/amazon-corretto.md

```diff
@@ -37,50 +37,50 @@ releases:
 -   releaseCycle: "19"
     eol: 2023-04-01
     latest: "19.0.2.7.1"
-    releaseDate: 2022-09-20
+    releaseDate: 2022-08-16
     latestReleaseDate: 2023-01-17
 
 -   releaseCycle: "18"
     eol: 2022-10-18
     latest: "18.0.2.9.1"
-    releaseDate: 2022-03-22
+    releaseDate: 2022-02-10
     latestReleaseDate: 2022-07-19
 
 -   releaseCycle: "17"
     lts: true
     eol: 2029-10-01
     latest: "17.0.6.10.1"
-    releaseDate: 2021-09-16
+    releaseDate: 2021-08-20
```

Comment:
So much `releaseDate` updates ! But this looks closer to the reality, at least for 17 (https://github.com/corretto/corretto-17/blob/release-17.0.6.10.1/CHANGELOG.md#corretto-version-1700351) and 11 (https://github.com/corretto/corretto-11/blob/release-11.0.18.11.1/CHANGELOG.md#initial-release-version-corretto-110291)